examples/coordinates/plot_sgr-coordinate-frame.py

```diff
@@ -5,12 +5,12 @@
 ==========================================================
 
 This document describes in detail how to subclass and define a custom spherical
-coordinate frame, as discussed in :ref:`astropy-coordinates-design` and the
-docstring for `~astropy.coordinates.BaseCoordinateFrame`. In this example, we
-will define a coordinate system defined by the plane of orbit of the Sagittarius
-Dwarf Galaxy (hereafter Sgr; as defined in Majewski et al. 2003).  The Sgr
-coordinate system is often referred to in terms of two angular coordinates,
-:math:`\Lambda,B`.
+coordinate frame, as discussed in :ref:`astropy:astropy-coordinates-design` and
```

Comment:
Just to make sure I understand: the changes here are because it is a .py file and someone might well copy this example to their own package, modify it, but have the links still render. Correct?

Reply:
Yes, for these cases.
I refrained from the `docs` since generally only the example codeblocks would be copied.